client/src/document/on-github.tsx

```diff
@@ -97,7 +97,9 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
       : "/mdn/content/issues/new";
   sp.set(
     "template",
-    locale !== "en-US" ? `page-report-${locale}.yml` : "page-report.yml"
+    locale !== "en-US"
+      ? `page-report-${locale.toLocaleLowerCase()}.yml`
```

Comment:
Let's just use `toLowerCase()`, as the locale is ASCII only:

```suggestion
      ? `page-report-${locale.toLowerCase()}.yml`
```

Reply:
Sorry about that, I haven't notice this 😢